int32 id

float32 x
float32 y

# enum: HOVER, CRUISE, AUTO
int16 move_mode

# enum: ABS, REL
int16 coord_system

float32 timeout
